src/psf/plugin.cc

```diff
@@ -141,6 +163,9 @@ bool PSFPlugin::read_tag(const char *filename, VFSFile &file, Tuple &tuple, Inde
     return true;
 }
 
+extern void setendless(int e);
+extern void setendless2(int e);
```

Comment:
These should be declared in peops/spu.h and peops2/spu.h.

Reply:
setlength() and setlength2() aren't in spu.h. They are externed when needed, which is why I followed the same practice.

https://github.com/audacious-media-player/audacious-plugins/blob/8d053afa9ee8327583b8521a14db481e61d883d5/src/psf/eng_spx.cc#L54

https://github.com/audacious-media-player/audacious-plugins/blob/8d053afa9ee8327583b8521a14db481e61d883d5/src/psf/eng_psf.cc#L69

If you think that is the correct way to do it, I have no objection and I'll add them to spu.h

Reply:
yeah please put it in spu.h

Reply:
The compiler can't find them when put in spu.h, and I'm not sure it'll be a good idea to include both spu.h files in plugin.cc.
Since it gets included directly, would eng_protos.h be more appropriate?

Reply:
I fixed up the existing extern declarations.  peops/spu.h and peops2/spu.h are now included from plugin.cc, so please put the declarations in those headers.

Reply:
Thank you. I fixed the declarations